spinedb_api/db_mapping_base.py

```diff
@@ -398,10 +433,19 @@ def find_item(self, item, skip_keys=(), fetch=True):
         return self.find_item_by_unique_key(item, skip_keys=skip_keys, fetch=fetch)
 
     def find_item_by_id(self, id_, fetch=True):
+        if id_ > 0:
+            try:
+                id_ = self.id_map.item_id(id_)
+            except KeyError:
+                if fetch:
+                    self._db_map.do_fetch_all(self._item_type)
+                    try:
+                        id_ = self.id_map.item_id(id_)
+                    except KeyError:
+                        return {}
+                else:
+                    return {}
```

Comment:
Previously, `_MappedTable.get()` would do the id translation via the (now unused) `self._temp_id_by_db_id`. I believe that was quite nice. This is almost the same, but now `_MappedTable.get()` doesn't do the id translation which may have some undesired consequences?

Reply:
You are right that `_MappedTable.get()` should do the id translation. I am not sure how it ended up in `find_item_by_id()`. I will restore `get()` back to its former glory.